packages/generator/src/edmx-to-vdm/common/complex-type.ts

```diff
@@ -118,9 +118,9 @@ export function getTypeMappingComplexProperties(
 
   if (isEnum) {
     return {
-      edmType: 'Edm.Enum',
+      edmType: 'Edm.String',
```

Comment:
Is this correct?

Reply:
The goal is to get rid of the `Edm.Enum`. Actually, the `edmType` of the enum situation is not useful.
If we don't want to make it optional, I guess `Edm.String` is the type fits well.

Reply:
Generally our API is ugly because we have edmType for all properties, even if they are not edm properties, but well...
If I am not mistaken this type is not used anymore, is it? So should we use anything here? Maybe assign undefined or null or something like that? I still find it confusing to have "Edm.String" here.